File: TCC/app/src/main/java/com/example/quemtocahoje/POJO/Banda.java
package com.example.quemtocahoje.POJO;

import java.util.List;

public class Banda {

    private Integer idBanda;
    private String nome;
    private String generoBanda;
    private String dataCriacao;
    private Arquivo arquivo;
    private Convite convite;
    private Evento evento;
    private List<Musico> musicos = null;

    public Integer getIdBanda() {
        return idBanda;
    }

    public void setIdBanda(Integer idBanda) {
        this.idBanda = idBanda;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getGeneroBanda() {
        return generoBanda;
    }

    public void setGeneroBanda(String generoBanda) {
        this.generoBanda = generoBanda;
    }

    public String getDataCriacao() {
        return dataCriacao;
    }

    public void setDataCriacao(String dataCriacao) {
        this.dataCriacao = dataCriacao;
    }

    public Arquivo getArquivo() {
        return arquivo;
    }

    public void setArquivo(Arquivo arquivo) {
        this.arquivo = arquivo;
    }

    public Convite getConvite() {
        return convite;
    }

    public void setConvite(Convite convite) {
        this.convite = convite;
    }

    public Evento getEvento() {
        return evento;
    }

    public void setEvento(Evento evento) {
        this.evento = evento;
    }

    public List<Musico> getMusicos() {
        return musicos;
    }

    public void setMusicos(List<Musico> musicos) {
        this.musicos = musicos;
    }

}

File: ApiQuemTocaHoje/ApiQuemTocaHoje/Models/Banda.cs
using System;
using System.Collections.Generic;

namespace ApiQuemTocaHoje.Models
{
    public class Banda
    {
        public Banda()
        {
            //this.Arquivo = new HashSet<Arquivo>();
            //this.Convite = new HashSet<Convite>();
            //this.Evento = new HashSet<Evento>();
            this.Musicos = new HashSet<Musico>();
        }

        public int IdBanda { get; set; }
        //public int NumIntegranteBanda { get; set; }
        public string Nome { get; set; }
        public string GeneroBanda { get; set; }
        public DateTime DataCriacao { get; set; }


        public Arquivo Arquivo { get; set; }

        public Convite Convite { get; set; }

        public Evento Evento { get; set; }

        public ICollection<Musico> Musicos { get; set; }



        //public int IdBanda { get; set; }
        //public string Nome { get; set; }
        //public string Descricao { get; set; }
        //public List<string> Generos { get; set; }
        //public DateTime DataCriacao { get; set; }
        //public int IdArquivo { get; set; }
        //public Arquivo Arquivo { get; set; }

        //public virtual ICollection<Musico> Musicos { get; set; }

    }
}